source/development_process/index.rst

```diff
@@ -85,12 +85,25 @@ Determine Requirements
 
 Setting Up the Project
 ----------------------
-- Create the git repo
-- Create the issue tracker
+
+**Create the git repo**: Unless you have special permission in the `OSL
+organization github <https://github.com/osuosl>`_, you will probably
+need to ask a full-time employee to make the repository for you.  We
+want all the repositories for the OSL to be owned by the OSL, so don't
+make a repo under your own account! Once the repo is created you'll
+want to use ``$ git clone <SSH URI>`` to start working on the project.
+**Create the Issue Traker**: Often times we will simply use the github
```

Comment:
All future projects will use the Redmine issue tracker. Describe how to create a new project on redmine.